### PR TITLE
Add density restrictions for C & M zones in LA

### PR DESCRIPTION
**Pull request by ryangomba**
I'm not sure if this change correct, but right now I am interpreting that you can build residential in C2 without any height or density limits, without needing a conditional review / permit.

### CODE REVIEW

**Comment by ryangomba** [comment]
(I'm not respecting the mapping to R4, which is in the notes, but not encoded in any formal way)

**Comment by ryangomba** [comment]
Annie tells me LA does indeed let you build residential in commercial zones, without conditional permits?

If that's the case, this diff shouldn't be merged, but we should come up with a strategy for encoding those residential mappings so those height / density constraints can be used for analysis.

**Comment by mattwigway** [comment]
So is the idea that anything you can do in an R4 you can also do in a C zone?

**Comment by hunterowens** [comment]
@mattwigway C == R4 for Res Uses

**Review by hunterowens — COMMENTED** [review]
I believe it should be 1,1 for SingleFamily and Multifamily to account for changes to specfiles. Right @mattwigway ?

**Comment by ryangomba** [comment]
Changed all 0,1 -> 1,1 for LA

File: src/zoning/specs/los-angeles.csv [hunk 1]
```diff
@@ -50,25 +50,25 @@ R1,1,0,45,5000,1,5000,50,,,,,2,,,,,0-20,20,0-50,10,15,,
...
+RD4,1,1,45,400,,8000,60,,,,,1-3,,,,,15,,5-10,10,15,,
+RD5,1,1,45,5000,,10000,70,,,,,1-3,,,,,20,,10,,25,,
+RD6,1,1,45,6000,,12000,70,,,,,1-3,,,,,20,,10,,25,,
 RE11,1,0,45,11000,1,11000,70,,,,,2,,,,,25-inf,20,10,,,25,
 RE15,1,0,45,15000,1,15000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE20,1,0,45,20000,1,20000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE40,1,0,45,40000,1,40000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE9,1,0,45,9000,1,9000,60,,,,,2,,,,,25-inf,20,10,,,25,
-RMP,0,1,45,20000,1,,80,,,,,2,,,,,0-25,25,10,,25,25,Mobile Home Park
+RMP,1,1,45,20000,1,20000,80,,,,,2,,,,,0-25,25,10,,25,25,Mobile Home Park
```
Comment (mattwigway) [inline src/zoning/specs/los-angeles.csv]: Are we considering mobile home parks to be multifamily housing here?

Comment (ryangomba) [reply src/zoning/specs/los-angeles.csv]: Yes... should that not be true? It's for the full park, right?

**Pull request closed without merging**